mailchimp3/entities/landingpageaction.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+"""
+The Landinge Page actions API endpoint
+
+Documentation: http://developer.mailchimp.com/documentation/mailchimp/reference/landing-pages/
```

Comment:
Documentation:  There is a shorter form for the documentation link, please use `https://mailchimp.com/developer/reference/landing-pages/`